core-bundle/src/Resources/contao/modules/ModuleQuicknav.php

```diff
@@ -130,13 +130,6 @@ protected function getQuicknavPages($pid, $level=1, $host=null)
 				$objSubpage->domain = $host;
 			}
 
-			// Hide the page if it is not protected and only visible to guests (backwards compatibility)
-			if ($objSubpage->guests && !$objSubpage->protected && $isMember)
```

Comment:
this should be a separate PR (including `tl_page.guests` and migration)

Reply:
Reverted in 965ab6aa605345f3e860857032cebdeb8a3bdd8f.